juju1/state/migration_export.go

```diff
@@ -1640,6 +1655,10 @@ func (e *exporter) storagePools() error {
 		return errors.Annotate(err, "listing pools")
 	}
 	for _, cfg := range poolConfigs {
+		if e.model.Config()["type"].(string) != "ec2" && cfg.Name() == "ebs-ssd" {
```

Comment:
I don't understand why you would filter out ebs-ssd specifically. What's special about it?

Reply:
In 1.25 the settings are created when the ebs provider is registered, even though this is a MAAS environment. Trying to import it into the 2.2.3 target fails because the provider isn't registered in 2 anymore (there was a change at some point to stop the automatic creation of the settings). I'm not sure this is the right way to handle it, but I think we do need to filter it out somehow.